config.toml.sample

```diff
@@ -23,3 +23,8 @@ ssl_mode = "disable"
 max_open = 25
 max_idle = 25
 max_lifetime = "300s"
+
+# Prometheus Metrics.
+[metrics.prometheus]
+enabled = true
+path = "/metrics"
```

Comment:
Perhaps a default value should be `/api/metrics` to conform with existing URI pattern

Reply:
I'm curious of your thoughts. Prometheus [best practice](https://prometheus.io/docs/introduction/first_steps/#configuring-prometheus) expects this path to be at `/metrics` and expects it to be available unauthenticated.

This is convenient and makes a lot of sense in environments like Kubernetes where reverse-proxying and isolating resources is trivial _(within the realm of you running Kubernetes)_.

I don't get a sense of what the average Listmonk deployment is like, so that might not be a good idea depending on our target audience. Especially the part where metrics are available unauthenticated.

Reply:
Path should not be configurable at all. Best to stick to one universal metrics path in all listmonk installations to avoid future confusion (and support burden).